app/controllers/webhooks/issue_updates_controller.rb

```diff
@@ -0,0 +1,23 @@
+# This controller listens for Github pull request webhook events,
+# for which the "Issues" event type must be enabled in GitHub.
+# See https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#issues
+module Webhooks
+  class IssueUpdatesController < BaseController
```

Comment:
To get this working, remember to set a global webhook that listens for the "Issues" event type and points to `/webhooks/issue_updates`